pkg/addon/parameters.go

```diff
@@ -3,76 +3,72 @@ package addon
 import (
 	"context"
 	"fmt"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	"regexp"
 	"strconv"
 
 	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
-	rhmiresources "github.com/integr8ly/integreatly-operator/pkg/resources/rhmi"
+	l "github.com/integr8ly/integreatly-operator/pkg/resources/logger"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
 	QuotaParamName      = "addon-managed-api-service"
 	TrialQuotaParamName = "trial-quota"
+	ManagedAPIService   = "managed-api-service"
+	RHMI                = "rhmi"
+	DefaultSecretName   = "addon-managed-api-service-parameters"
```

Comment:
hey Maks, any reason for exporting this const?

Reply:
Right now no, but there is only one more place we are using those and if anywhere in `pkg` we would need to use them we will have an import cycle. Made them public for future usage. Could go with private if you have a strong opinion 🤔

Reply:
If you decide to fix typos I would update this value too.

I understand that thinking of supporting future RHOAM versions might be good to have, but, have you ever heard 
`Don't cross the bridge till you come to it` sentence?